.github/workflows/build.yml

```diff
@@ -14,42 +14,67 @@ env:
   BUILD_TYPE: Release
 
 jobs:
+  # Do a plain build, like we would expect a user to do it on their end.
```

Comment:
Nitpick: I don't expect a user to install Homebrew to build Furnace :p

Reply:
I'm open for a better naming suggestion ;p